server/server/src/lib.rs

```diff
@@ -41,331 +38,218 @@ pub mod static_files;
+        (Some(database_sync_settings), Some(yaml_sync_settings)) => {
+            if database_sync_settings.core_site_details_changed(&yaml_sync_settings) {
+                info!("Sync settings in configurations dont match database");
+                info!("Checking sync credentails are for the same site..");
+                service_provider
+                    .site_info_service
+                    .request_and_set_site_info(&service_provider, &yaml_sync_settings)
+                    .await
+                    .unwrap();
+                info!("Checking sync credentails are for the same site..done");
+            }
+            service_provider
+                .settings
+                .update_sync_settings(&service_context, &yaml_sync_settings)
+                .unwrap();
+            false
         }
+        (None, Some(yaml_sync_settings)) => {
+            info!("Sync settings in configurations and not in database");
+            info!("Checking sync credentails..");
+            // If fresh sync settings provided in yaml, check credentials against central server and save them in database
+            service_provider
+                .site_info_service
+                .request_and_set_site_info(&service_provider, &yaml_sync_settings)
+                .await
+                .unwrap();
+            info!("Checking sync credentails..done");
+            service_provider
+                .settings
+                .update_sync_settings(&service_context, &yaml_sync_settings)
+                .unwrap();
+            true
+        }
+        _ => false,
     };
 
-    if db_settings.is_none() && config_settings.sync.is_some() {
-        service_provider
-            .site_info_service
-            .request_and_set_site_info(&service_provider, config_settings.sync.as_ref().unwrap())
-            .await
-            .unwrap();
-    }
-
-    // Final settings:
-    let mut settings = config_settings;
-    settings.sync = Some(sync_settings.clone());
-    let site_id = service_provider
-        .site_info_service
-        .get_site_id(&service_context)
+    // CREATE GRAPHQL SCHEMA
+    let is_operational = service_provider
+        .sync_status_service
+        .is_initialised(&service_context)
         .unwrap();
-
-    let auth_data = auth_data(
-        &settings.server,
-        token_bucket.clone(),
-        token_secret.clone(),
-        &certificates,
+    info!(
+        "Creating graphql schema in {} mode..",
+        match is_operational {
+            true => "operational",
+            false => "initialisation",
+        }
     );
+    let graphql_schema = Data::new(GraphqlSchema::new(
+        GraphSchemaData {
+            connection_manager: Data::new(connection_manager),
+            loader_registry: Data::new(LoaderRegistry { loaders }),
+            service_provider: service_provider.clone(),
+            settings: Data::new(settings.clone()),
+            auth,
+            // TODO what what this like before ?
```

Comment:
Lol, oki, so what it looks like it was only used to not have logger in self requester (since there was a common method to create schema for self requester and normal operational, so i removed, and simplified a little bit: a5e7e68a9372616a230be51045e5a096c8e5fdb4